app/scripts/views/download/download-controller.js

```diff
@@ -57,9 +57,18 @@
         function csvTransform(data) {
             var headers = _.keys(_.omit(data[0], '$$hashKey'));
 
+            var quoteReplaceRe = /"/g;
+
             var csvString = headers.join(',') + '\n';
             angular.forEach(data, function (row) {
-                csvString += _.values(_.omit(row, '$$hashKey')).join(',') + '\n';
+                csvString += _(row).omit('$$hashkey').values().map(function (value) {
+                    if (_.isString(value)) {
+                        // Need to quote strings, and convert double quotes to single quotes
+                        return '"' + value.replace(quoteReplaceRe, '\'') + '"';
```

Comment:
According to http://www.csvreader.com/csv_format.php:

> In an Excel escaped CSV file, in fields containing a double quote, the double quote must be escaped by replacing the single double quote with two double quotes.

But single quotes are fine.